# Internal patch.
# Remove dependency on whether or not OPENSSL_NO_DEPRECATED
# is raised or not.
# Not for upstream. This version of libtorrent is much too old anyway.
--- libtorrent-0.12.2/src/utils/diffie_hellman.cc	2015-04-09 07:53:18.395899128 -0700
+++ libtorrent-0.12.2/src/utils/diffie_hellman.cc	2015-04-09 07:59:11.807442371 -0700
@@ -100,7 +100,7 @@
 			{
 			BN_init(&local_prk);
 			prk = &local_prk;
-			BN_with_flags(prk, priv_key, BN_FLG_EXP_CONSTTIME);
+			BN_with_flags(prk, priv_key, BN_FLG_CONSTTIME);
 			}
 		else
 			prk = priv_key;
